src/graph/GoExecutor.cpp

```diff
@@ -681,7 +681,7 @@ std::vector<VertexID> GoExecutor::getDstIdsFromRespWithBackTrack(const RpcRespon
     // 7  ,  6
     // Will mistake lead to 7->6 if insert edge(dst, src) one by one
     // So read all roots of current step first , then insert them
-    std::multimap<VertexID, VertexID> backTrace;
+    std::multiset<std::pair<VertexID, VertexID>> curBackTrace;
```

Comment:
Why not use unordered_set?

Reply:
Using unorderd_set here looks more effecient, I have modified it, please review.